eq-tests/scripts/run_smoke_test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# It is assume your system is steup as stated in the READMe.md
+
+export RABBITMQ='infrataster-rabbitmq1.eq.ons.digital'
+export SURVEYRUNNER='infrataster-surveys.eq.ons.digital'
+
+cd ..
```

Comment:
This feels fragile - if you're in the root directory of the project and run the script as ./scripts/run_smoke_test.sh, this fails as it's based on the location you're executing the script not the location of the script. http://stackoverflow.com/questions/59895/can-a-bash-script-tell-what-directory-its-stored-in is a good example of getting the path of the script from which you can then run `$DIR/../bundle exec rspec` to make this robust.